src/app/containers/Video/index.jsx

```diff
@@ -48,11 +49,21 @@ const VideoContainer = ({ blocks }) => {
     aresMediaBlock.model,
   );
   const guidance = deepGet(['warnings', 'short'], version);
+
   const id = `mp#${pid}`;
+
+  const statsObject = { destination: statsDestination };
+
+  if (subType === 'clip') {
+    statsObject.clipPID = pid;
+  } else if (subType === 'episode') {
```

Comment:
We currently have no working examples of Test episode assets - this is why we don't have fixture data for this, for testing.